docs/DeveloperGuide.md

```diff
@@ -432,7 +432,18 @@ _{more aspects and alternatives to be added}_
 - Modify the format of `DATE` to be `d-M-yyyy` and `DATETIME` to be `d-M-yyyy`T`H:mm` instead.
 - This allows FINDvisor to accept both single and double-digits day, month and hour values as valid `DATE` and `DATETIME` values and would not require users to pad these single digit values with a leading zero.
 
-### 3. Show warning to user when scheduling an overlapping meeting
+### 5. Notfiy user if data file is invalid
+**Current Implementation**: 
+
+- If the data file cannot be parsed due to invalid fields, FINDvisor will discard all data and start with an empty data file at the next run.
+- No error message is displayed to the user when this occurs and users are not informed of the data loss.
+
+**Proposed Enhancement**: 
+
+- Display a message in the Command Result Box if the data file cannot be parsed on startup.
+- For example, `Data file (<file_location>) could not be loaded!` will be shown when the data file is invalid, where `file_location` is the location of the current referenced data file.
```

Comment:
We should specify what we do with the invalid data file too, either still discard it or move it to somewhere.